docs/source/cuda/cuda_array_interface.rst

```diff
@@ -69,13 +69,213 @@ The following are optional entries:
 
   If ``None`` then all values in **data** are valid. All elements of the mask
   array should be interpreted only as true or not true indicating which
-  elements of this array are valid. This has the same definition as ``mask``
-  in the `numpy array interface`_.
+  elements of this array are valid. This has the same definition as *mask* in
+  the `numpy array interface`_.
 
   .. note:: Numba does not currently support working with masked CUDA arrays
-            and will raise a ``NotImplementedError`` exception if one is passed
+            and will raise a `NotImplementedError` exception if one is passed
             to a GPU function.
 
+- **stream**: ``integer``
```

Comment:
At some point in the future we might want to allow producers to (optinally) include a stream event with the CAI, where the stream event is enqueued immediately after the buffer is ready.  This unblocks some other use cases.

CC @bas-aarts

Reply:
+1 to optionally allowing a stream event to wait on in the future

Reply:
A note has been added to suggest this might be added in future.